packages/expo-calendar/android/src/main/java/expo/modules/calendar/CalendarModule.java

```diff
@@ -1,62 +1,62 @@
-// Copyright 2015-present 650 Industries. All rights reserved.
-
-package versioned.host.exp.exponent.modules.api;
+package expo.modules.calendar;
 
 import android.Manifest;
 import android.content.ContentResolver;
 import android.content.ContentUris;
 import android.content.ContentValues;
+import android.content.Context;
 import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.database.Cursor;
 import android.net.Uri;
 import android.os.AsyncTask;
+import android.os.Bundle;
 import android.provider.CalendarContract;
 import android.text.TextUtils;
-import android.database.Cursor;
 import android.util.Log;
 
-import com.facebook.react.bridge.Promise;
-import com.facebook.react.bridge.ReactApplicationContext;
-import com.facebook.react.bridge.ReactContext;
-import com.facebook.react.bridge.ReactContextBaseJavaModule;
-import com.facebook.react.bridge.ReactMethod;
-import com.facebook.react.bridge.ReadableArray;
-import com.facebook.react.bridge.ReadableMap;
-import com.facebook.react.bridge.ReadableType;
-import com.facebook.react.bridge.WritableArray;
-import com.facebook.react.bridge.WritableMap;
-import com.facebook.react.bridge.WritableNativeArray;
-import com.facebook.react.bridge.WritableNativeMap;
-import com.facebook.react.bridge.Dynamic;
-
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.util.ArrayList;
 import java.util.Calendar;
-import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.TimeZone;
 
-import host.exp.exponent.kernel.ExperienceId;
-import host.exp.expoview.Exponent;
-import versioned.host.exp.exponent.modules.ExpoKernelServiceConsumerBaseModule;
+import expo.core.ExportedModule;
+import expo.core.ModuleRegistry;
+import expo.core.Promise;
+import expo.core.arguments.ReadableArguments;
+import expo.core.interfaces.ExpoMethod;
+import expo.core.interfaces.ModuleRegistryConsumer;
+import expo.interfaces.permissions.Permissions;
 
-public class CalendarModule extends ExpoKernelServiceConsumerBaseModule {
+public class CalendarModule extends ExportedModule implements ModuleRegistryConsumer {
   private static final String TAG = CalendarModule.class.getSimpleName();
 
-  private ReactContext reactContext;
+  private Context mContext;
```

Comment:
You could just use `getContext()`, but if you don't want to change that file only for that I think that's ok with me